modules/readline/8.2/overlay/BUILD.bazel

```diff
@@ -0,0 +1,132 @@
+load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+
+licenses(["restricted"])  # GPL v3
+
+cc_library(
+    name = "readline",
+    srcs = glob(
+        ["*.c"],
+        exclude = ["*_keymap.c"],
+    ) + [
+        "readline/config.h",
+    ],
+    hdrs = glob(["*.h"]) + [
+        "emacs_keymap.c",
+        "vi_keymap.c",
+    ],
+    copts = [
+        "-w",
+        "-DHAVE_CONFIG_H",
+        "-DRL_LIBRARY_VERSION='\"8.2\"'",
+    ],
+    defines = [
+        "READLINE_HAVE_CONFIG_H",
+    ],
+    includes = [
+        "readline",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":prefixed_readline_header",
```

Comment:
Could this be in `implementation_deps` or do we want to propagate both ways of accessing the headers?

Reply:
I believe that's the desire here, to support this include path.